woocommerce/google-listings-and-ads
# Attribute Mapping - Spotlight Component

### Changes proposed in this Pull Request:

This PR integrates TourKit in Attribute Mapping page and in the navigation 

### Screenshots:

<img width="1173" alt="Screenshot 2022-12-14 at 18 07 37" src="https://user-images.githubusercontent.com/5908855/207579721-d89e22cf-23e8-4f7a-a66b-b0fe4a2c0c72.png">

<img width="1180" alt="Screenshot 2022-12-14 at 18 07 51" src="https://user-images.githubusercontent.com/5908855/207580042-271d9002-2577-4798-a2c5-40a1e828bd96.png">



### Detailed test instructions:


1. Enter into the GLA Dashboard. You will see the tour over Attribute mapping tab.
2. Navigate over different tabs. (Except Attribute Mapping) and see how the tour persists.
3. Navigate to Attribute Mapping. It doesn't show anymore the tour for the navigation. Instead It shows another tour for the feature.
4. Navigate back to other tab. See how tour for the navigation appears again.
5. Click on close or "Got it" and see how the tour disappears.
6. Refresh the page and see how it won't show the tour.
7. Go to Attribute Mapping tab. See the tour. Click on Got it and see how it disappears.
8. Refresh the page and see how it won't show the tour.


### Additional details:

⚠️  Test four the tours are not implemented in this PR since they are still creating some issues and needs further investigation. It will be wrangled in cooldown.

### Changelog entry

> Add - Interactive discovery message for Attribute mapping feature.
